js/data.js

```diff
@@ -344,12 +388,23 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze", "Frank Feinbube", "Uwe Hentschel"],
         nameLV: "Eingebettete Betriebssysteme"
     },
+    eingebetteteBetriebssystemeWithITSE: {
+        kurz: "Embedded<br />Systems (mit ITSE)",
+        kennung: [IST, OSIS, SAMT, ITSE],
```

Comment:
Here it says, SAMT is possible: http://hpi.de/studium/lehrveranstaltungen/it-systems-engineering/lehrveranstaltung/course/2015/eingebettete_betriebssysteme-2.html